docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md

```diff
@@ -165,6 +165,25 @@ If you don't provide the target YugabyteDB database name during import, yb-voyag
 CREATE DATABASE target_db_name;
 ```
 
+{{<note title="Important">}}
```

Comment:
Can we shift this note just before the "create the target database" section ? Like we should let user know this while preparing target cluster.